pkg/common/oras/authprovider/azure/azureworkloadidentity_test.go

```diff
@@ -372,7 +372,7 @@ func TestAzureWIEnabled_ExpectedResults(t *testing.T) {
 func TestGetEarliestExpiration(t *testing.T) {
 	var aadExpiry = time.Now().Add(12 * time.Hour)
 
-	if getACRExpiryIfEarlier(aadExpiry) == aadExpiry {
+	if time.Time.Equal(aadExpiry, getACRExpiryIfEarlier(aadExpiry)) {
```

Comment:
nit(test): this seems not right. should it be something like `aadExpiry.Equal(getACRExpiryIfEarlier(aadExpiry))`?

Reply:
good call, fixed!